src/main/java/com/google/cloud/solutions/autotokenize/encryptors/GcpKmsClearTextKeySetExtractor.java

```diff
@@ -48,7 +54,7 @@ public String get() throws GeneralSecurityException, IOException {
     var tinkKeySet =
         KeysetHandle.read(
             JsonKeysetReader.withString(tinkKeySetJson),
-            new GcpKmsClient().withDefaultCredentials().getAead(mainKeyKmsUri));
+            new GcpKmsClient().withCredentials(credential).getAead(mainKeyKmsUri));
```

Comment:
This one doesn't work when using accessToken. It works on my local machine when using default credentials, but no such thing is present in Juptyter

Reply:
Fixed